.github/workflows/crypto-test-harness.yml

```diff
@@ -0,0 +1,35 @@
+name: Crypto algorithm implementation testing pipeline
+
+on:
+  pull_request:
+    branches: [ "cf-zeta" ]
+  workflow_dispatch:
+
+jobs:
+  crypto_testing_job:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v3
+    - name: build kernel & modules
+      run: |
+        cp zeta/test-artifacts/config-um .config
+        make olddefconfig ARCH=um
+        make ARCH=um all
```

Comment:
we probably want to add `-j <numcpus>` here to make it faster

Reply:
Added in latest version.